app/src/component/relatedContent/index.js

```diff
@@ -14,7 +14,7 @@ const select = (state, props) => ({
 });
 
 const perform = dispatch => ({
-  search: query => dispatch(doSearch(query, 10, undefined, true)),
+  search: query => dispatch(doSearch(query, 20, undefined, true)),
```

Comment:
So, funny story, if I set a different value from 20 here, no results are displayed for related content. The `makeSelectRecommendedContentForUri` selector needs to be updated to be able to handle different result counts. Something to be considered in the future.

Reply:
Ah that's not very good. I will look into it.